docs/next.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = withTypescript({
   webpack: (config, options) => {
     // Alias @material-ui/core peer dependency imports form the following modules to our sources.
     config = withTM({
-      transpileModules: ['notistack', '@material-ui/pickers', 'material-table'],
+      transpileModules: ['notistack', '@material-ui', 'material-table'],
```

Comment:
It's still included. This will pick up every module under `@material-ui/`

Reply:
I was assuming that `@material-ui/pickers` is the only module we need to pick. So to make sure I understand correctly, this is a change in case, in the future, we have to handle a new, let's say `@material-ui/table` module?

Reply:
The aliases are now setup so that we use our packages via node_modules/ instead of packages/.

Might not be required but I think next had a problem when using source outside of its root. Will double check

Reply:
Confirmed that we need this. Can't alias to `../packages/material-ui/src`. We have to use `@material-ui/core/src` and tell `next` to transpile those modules.

Reply:
By using a node_modules/ alias we also disabled hmr. We need to use the ./packages/material-ui* alias to make hmr work. This required an additonal loader rule for webpack which means changing transpiled modules is no longer required.